source/help/getting-started/configuring-notifications.rst

```diff
@@ -63,14 +52,28 @@ Mobile push notifications appear on the lock screen of your mobile device if the
 Browser Tab Notifications
 ----------------------------------------
 
-If Mattermost is open in a browser tab, the favicon updates to notify you of unread messages (\*) and
-a count of mentions or Direct Messages. Browser tab notifications are available on Firefox and Chrome.
+If Mattermost is open in a browser tab, the favicon updates to notify you of unread messages (\*) and a count of mentions or Direct Messages. Browser tab notifications are available on Chrome, Edge, Firefox, Safari.
 
 .. image:: ../../images/browser_notification.png
 
 Muting a Channel
 ----------------------------------------
 
-Channels in the sidebar are **bold** when there are unread messages in the channel and show a badge count if you are mentioned specifically. To mute a channel so it does not appear unread and you do not recieve notifications, click the channel name at the top of the page to access the **Channel Menu** > **Notification Preferences > Mute channel**.
+Channels in the sidebar are **bold** when there are unread messages in the channel and show a badge count if you are mentioned specifically. 
+
+When you mute a channel, you do not receive any notifications (desktop, email, or push) for any mentions (whether they are channel-wide to directed at you). However, if you are mentioned while the channel is muted the mention badge will be displayed in the sidebar with the channel displaying at reduced opacity.  
+
+To mute a channel, click the channel name at the top of the page and navigate to **Notification Preferences > Mute channel**, where you can edit the notification settings for that channel. You can also click on the channel name and select **Mute Channel**. To unmute it, click on the channel name and select **Unmute Channel**.
```

Comment:
Do we need to state both methods here or just the easiest one?

Reply:
Good catch, I've edited it to use the easiest one.